pom.xml

```diff
@@ -58,6 +58,16 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>com.github.ben-manes.caffeine</groupId>
+      <artifactId>caffeine</artifactId>
+      <version>2.7.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.github.ben-manes.caffeine</groupId>
+      <artifactId>guava</artifactId>
```

Comment:
This just looks like [trouble](https://jenkins.io/doc/developer/plugin-development/dependencies-and-class-loading/). Delete and just use Caffeine’s native interfaces.

Reply:
@jglick 
This is an adapter whose code is in the same repo as Caffeine and the version is the same. 
I used to it to because it keeps the same contract, minimizing changes. 

I've update the pom.xml to make this clearer.

Reply:
Run `mvn dependency:tree` to make sure it is not pulling in its own version of `guava`. If it is trying to, you can `<exclude>` that and use the version Jenkins has. But, again, it would be safer to just not use this adapter.

Reply:
Ah, that would be why the change to the test.